src/modelinstance.cpp

```diff
@@ -563,9 +563,8 @@ uint ModelInstance::getNumOfParallelInferRequestsUnbounded(const ModelConfig& mo
         // nireq is set globally for all models in ovms startup parameters
         return ovmsConfig.nireq();
     }
-    std::string key = METRIC_KEY(OPTIMAL_NUMBER_OF_INFER_REQUESTS);
     try {
-        numberOfParallelInferRequests = compiledModel->get_property(key).as<unsigned int>();
+        numberOfParallelInferRequests = compiledModel->get_property(ov::optimal_number_of_infer_requests);
     } catch (const ov::Exception& ex) {
         SPDLOG_WARN("Failed to query OPTIMAL_NUMBER_OF_INFER_REQUESTS with error {}. Using 1 nireq.", ex.what());
```

Comment:
It would be good to fix logger here as well to use SPDLOG_LOGGER_WARN(modelmanager_logger,